### ISSUE TITLE
Editorial: WCAG 2.1 and 2.2: Guidelines no longer provide links to the understanding for the guideline

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#2769]
Not sure if this was an intentional change or not, but the links that used to exist in WCAG 2.0 that go to the understanding for each guideline no longer exist from the WCAG 2.1 Recommendation or WCAG 2.2 Candidate Recommendation. This means there's many more steps to get to the understanding for each guideline. 

Example from WCAG 2.0:
<img width="1076" alt="Screen shot shows heading for Guideline 1.2 with Understanding Guideline 1.2 link in the same callout box" src="https://user-images.githubusercontent.com/5894100/200424974-79460420-e1df-4460-a9b8-b202298b87e7.png">

Example from WCAG 2.2:
<img width="704" alt="Screen Shot shows heading for Guideline 1.2 with no understanding link" src="https://user-images.githubusercontent.com/5894100/200426467-b6a916fc-c3d0-41eb-8630-213f4798ccac.png">

To read the [Guideline 1.2 understanding document](https://www.w3.org/WAI/WCAG22/Understanding/time-based-media), you've got to use one of the following more involved methods to find it: 
1. Search for the link to the full understanding document (provided you know what to search for) and open it, find and then click the guideline you want to read about
2. From WCAG 2.2, link to an SC's understanding (has to be an SC listed under a particular guideline) and then use the link to the guideline for that SC
3. From WCAG 2.2, link to an SC's understanding, then use the link to go to the table of Contents, scroll or search for the guideline, and click the link to get to the understanding for the guideline.

**Comment by GreggVan** [comment]
Wow 

Those links are REALLY valuable
I hope that it was just a production problem that cause them to disappear.

Most people will not know those documents even exist —  much less be able to find them.  (And a MUCH higher cognitive load ) 

Can we get them back?    Understanding and How to meet links. 

gregg

------------------------------
Gregg Vanderheiden
***@***.***

**Comment by maryjom** [comment]
@GreggVan The links are there for the SC's, just missing for the Guidelines so there's no How To link for that.

**Comment by bruce-usab** [comment]
In a similar vein, I would like to ask that the 2.2 formatting include the thirteen Guidelines in a state-alone listing some how or another.  Probably the only aspect I miss with the 2.0 formatting is that, when teaching, I could put the [Table of Contents](https://www.w3.org/TR/WCAG20/#contents) on the screen and point out that **_the Guidelines are a top-<del>ten</de> <ins>twelve</ins> list_**!

We have lost this "WCAG at a Glance" feature which was part of the 2.0 formating.
